src/main/java/com/codejoust/main/dto/game/GameMapper.java

```diff
@@ -84,9 +85,22 @@ public static SubmissionDto submissionToDto(Submission submission) {
             return null;
         }
 
+        List<SubmissionResultDto> notHidden = new LinkedList<>();
+
+        if (submission.getResults() != null) {
```

Comment:
I still think it'd be nice to allow users to distinguish between wrong answer or runtime error - maybe instead of not sending submission results here, could you try clearing the `SubmissionResultDto` `input`, `console`, `userOutput`, and `correctOutput` fields? And then the `error` field can be used to check if an error occurred or otherwise it was a wrong answer. 

A side note, I think it might be good to clear the `correctOutput` field for all SubmissionResultDtos (cause we don't want people to snoop through those), and also, in the GameDto, making sure that the list of problems do not include the output for any test case and do not include the input for hidden test cases